pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: "Migrating from GOV.UK PaaS"
+---
+
+# Migrating from GOV.UK PaaS
+
+## Last updated
+
+- 1st November 2022 - first published
```

Comment:
1 November - house style

Reply:
I will normalise all date references as discussed and make sure quarterly references are in the format QQ YYYY